src/Tec/Plugin.php

```diff
@@ -144,43 +148,32 @@ function option_filter( $fields ) {
 			'3|day'     => esc_html__( '3 days', 'the-events-calendar' ),
 			'1|week'    => esc_html__( '1 week', 'the-events-calendar' ),
 			'2|week'    => esc_html__( '2 weeks', 'the-events-calendar' ),
-			'1|month'   => esc_html__( '1 month', 'the-events-calendar' ),
-			'3|month'   => esc_html__( '3 months', 'the-events-calendar' ),
-			'6|month'   => esc_html__( '6 months', 'the-events-calendar' ),
-			'9|month'   => esc_html__( '9 months', 'the-events-calendar' ),
-			'1|year'    => esc_html__( '1 year', 'the-events-calendar' ),
-			'2|year'    => esc_html__( '2 years', 'the-events-calendar' ),
-			'3|year'    => esc_html__( '3 years', 'the-events-calendar' ),
+			'1'         => esc_html__( '1 month', 'the-events-calendar' ),
+			'3'         => esc_html__( '3 months', 'the-events-calendar' ),
+			'6'         => esc_html__( '6 months', 'the-events-calendar' ),
+			'9'         => esc_html__( '9 months', 'the-events-calendar' ),
+			'12'        => esc_html__( '1 year', 'the-events-calendar' ),
+			'24'        => esc_html__( '2 years', 'the-events-calendar' ),
+			'36'        => esc_html__( '3 years', 'the-events-calendar' ),
 		];
 		$fields['delete-past-events']['options'] = $new_values;
 		$fields['trash-past-events']['options']  = $new_values;
 
 		return $fields;
 	}
 
-	/**
-	 * Triggering the rescheduling of crons
-	 *
-	 * @return void
-	 */
-	function reschedule_crons() {
-		$this->reschedule_trash_or_del_event_cron( 'tribe_trash_event_cron' );
-		$this->reschedule_trash_or_del_event_cron( 'tribe_del_event_cron' );
-	}
-
 	/**
 	 * Rescheduling the crons handling the trashing and deleting.
 	 *
-	 * @param string $cron The slug of the cron.
+	 * @since 1.2.2 Adjust cron frequency calculation.
+	 *              Remove $cron parameter.
 	 *
-	 * @return void
+	 * @return string The frequency string how often the cron should run.
 	 */
-	function reschedule_trash_or_del_event_cron( $cron ) {
-		if ( 'tribe_trash_event_cron' == $cron ) {
-			$time = tribe_get_option( 'trash-past-events', 43200 );
```

Comment:
The default was twicedaily now it became once a month. 43200 is half a day in seconds